vars/main.yml

```diff
@@ -230,8 +236,20 @@ wal_e:
   - { option: "no_master",  value: "1" }
   - { option: "envdir",     value: "/etc/wal_e/envdir" }
   - { option: "use_iam",    value: "1" }
+wal_g:
+  - { option: "command",    value: "sudo -u postgres wal-g backup-fetch --datadir /val/lib/postgresql/data LATEST" }
```

Comment:
"sudo -u postgres" is superfluous here.

and it is necessary to check whether it will work if the variable on the PGDATA is specified:
```

  - { option: "command",    value: "wal-g backup-fetch --datadir {{ postgresql_data_dir }} LATEST" }
```